support-files/rpm/my.cnf

```diff
@@ -5,7 +5,7 @@
 [client-server]
 
 #
-# include all files from the config directory
+# include *.cnf from the config directory
```

Comment:
Are you sure that `.ini` files will never be read?
Can you please revise the commit message to say which code is reading the files? A quick `git grep` suggests that the `.ini` suffix only applies to Windows, but it looks like it might be possible to override that by some option. The relevant call appears to be `search_default_file()` in `mysys/my_default.c`.

Reply:
It is said in [documentation](https://mariadb.com/kb/en/configuring-mariadb-with-option-files/)
```
You can configure MariaDB to run the way you want by configuring the server with MariaDB's option files. The default MariaDB option file is called my.cnf on Unix-like operating systems and my.ini on Windows. Depending on how you've installed MariaDB, the default option file may be in a number of places, or it may not exist at all.
```
You can use `--defaults-file` to specify a file.
This patch is based for rpm and I assume problem is that  you used name without suffix `.cnf`. This is also covered in documentation
```
The !includedir directive can be used to include all .cnf files (and potentially .ini files) in a given directory. The option files within the directory are read in alphabetical order. See the Including Option File Directories section below for more information on this syntax.
```
I agree that we can fix the line to specify which files are from interest and to link to documentation if someone try without reading it first.

Reply:
> Are you sure that `.ini` files will never be read?

The docs say:
> All option file names must end in .cnf on Unix-like operating systems. On Windows, all option file names must end in .cnf or .ini.
https://mariadb.com/kb/en/configuring-mariadb-with-option-files/#including-option-file-directories

I'm assuming this file will never be used on Windows as it is contained in the `rpm/` directory and refers to `/etc`. But I did personally come across it on macOS which doesn't use rpm either. 🤷‍♂

> Can you please revise the commit message to say which code is reading the files? A quick `git grep` suggests that the `.ini` suffix only applies to Windows, but it looks like it might be possible to override that by some option. The relevant call appears to be `search_default_file()` in `mysys/my_default.c`.

I'll look at the code and update the commit message.

Thanks for reviewing!

Reply:
The relevant code appears to be

extension definition: https://github.com/MariaDB/server/blob/10.5/mysys/my_default.c#L69-L75

!includedir implementation: https://github.com/MariaDB/server/blob/10.5/mysys/my_default.c#L681-L715